github/Makefile

```diff
@@ -45,9 +45,9 @@ regenerate:
 
 export: image
 	$(info [$(YELLOW)*$(NORMAL)] Exporting docker image)
-	@printf "\n ---> Exporting Docker image to ./$(VENDOR)_$(NAME)_$(VERSION).tar\n"
-	@docker save $(VENDOR)/$(NAME):$(VERSION) | gzip > $(VENDOR)_$(NAME)_$(VERSION).tar
+	@printf "\n ---> Exporting Docker image to ./$(NAME).img\n"
+	@docker save $(VENDOR)/$(NAME):$(VERSION) | gzip > $(NAME)_$(VERSION).img
```

Comment:
This Makefile line is out of date, can you run `update-tools.sh` in the `tools/` directory, then regenerate the plugin?